bcauth/jinja2/account/email_confirmed.html

```diff
@@ -4,7 +4,6 @@
 {% block body_title %}{{ _("Confirm E-mail Address") }}{% endblock %}
 
 {% block content %}
-{% user_display confirmation.email_address.user as user_display %}
```

Comment:
I'm not sure either. My suspicion is that I didn't fully translate the original Django template:

https://github.com/pennersr/django-allauth/blob/master/allauth/templates/account/email_confirm.html

This template is used on the POST to the email confirmation view, such as http://localhost:8000/accounts/confirm-email/grxnjqegrsqrkheetfa3skkehv7kais0au88ueftyon4xm5gukd1a4h4knkym2yz/.

It requires having an email confirmation code, which could be setup in the admin, or by adding an email in the profile views. However, forms are broken due to CSRF token issues, so I can't manually test the effect.